src/core/jpx.js

```diff
@@ -57,6 +59,28 @@ class JpxImage {
       }
       const results = await WebAssembly.instantiate(this.#buffer, imports);
       return successCallback(results.instance);
+    } catch (reason) {
+      warn(`JpxImage#instantiateWasm: ${reason}`);
+
+      if (typeof PDFJSDev !== "undefined" && PDFJSDev.test("TESTING")) {
+        this.#wasmUrl ??= "/build/generic/web/wasm/";
+      }
+      const path =
+        (typeof PDFJSDev === "undefined" ? "../" : "") +
+        `${this.#wasmUrl}openjpeg_nowasm_fallback.js`;
```

Comment:
I don't think the pre-processor is "smart" enough to be able to concatenate strings, so I'd suggest just re-writing this as follows instead.
```suggestion
      const path =
        typeof PDFJSDev === "undefined"
          ? `../${this.#wasmUrl}openjpeg_nowasm_fallback.js`
          : `${this.#wasmUrl}openjpeg_nowasm_fallback.js`;
```